### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by brijeshjagani**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \ Pushing New FAQ

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [X] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 1]
```diff
@@ -6872,3 +6872,115 @@ Users can utilize the following API to retry a failed stage: [retryPipeline](htt
...
+#### What is the process for cloning autostopping rules in Harness?
+
+To clone an autostopping rule in Harness, follow these steps:
+
+Navigate to the AutoStopping Rules dashboard in Harness.
+Select the autostopping rule you want to clone.
+Click on the three-dot menu and choose "Clone".
+Provide a new name for the cloned rule.
+Review and adjust the configuration of the cloned rule as needed.
+Save the cloned rule.
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6885]: Can you make these steps a numbered list?

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 2]
```diff
@@ -6872,3 +6872,115 @@ Users can utilize the following API to retry a failed stage: [retryPipeline](htt
...
+
+#### How can I revert to the old UI in Harness?
+
+To switch back to the old UI in Harness, adjust the user preference settings in your Account Settings.
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6889]: Can we get more detail on how to navigate here in a "**Account Settings** > **User Preferences** > **Old UI**" style format?

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 3]
```diff
@@ -6872,3 +6872,115 @@ Users can utilize the following API to retry a failed stage: [retryPipeline](htt
...
+
+#### Is it possible to switch the Harness UI language to French?
+
+No, localization is currently not supported in the Harness Platform, including switching the UI language to French.
+
+#### Does the Community Edition of Harness support RBAC (Role-Based Access Control)?
+
+Yes, the Community Edition of Harness supports Role-Based Access Control (RBAC).
+
+#### How to Install Powershell on harness delegate?
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6899]:
```suggestion
#### How to Install Powershell on the Harness Delegate?
```

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 4]
```diff
@@ -6872,3 +6872,115 @@ Users can utilize the following API to retry a failed stage: [retryPipeline](htt
...
+name: INIT_SCRIPT
+  value: |
+    #!/bin/bash
+    sudo apt-get update
+    sudo apt-get install -y powershell
+```
+
+This script uses the apt-get package manager to install PowerShell on the delegate. You can modify the script to use a different package manager if needed. Once you have added the script to the delegate YAML file, apply the changes to the delegate using the kubectl apply command.
+For more details on how to create custom images delegate dockerfile
+https://developer.harness.io/docs/platform/delegates/manage-delegates/build-custom-images-delegate-dockerfile/
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6912]:
```suggestion
go to [Custom Images Delegate Dockerfile](]https://developer.harness.io/docs/platform/delegates/manage-delegates/build-custom-images-delegate-dockerfile/)
```

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 5]
```diff
@@ -6872,3 +6872,115 @@ Users can utilize the following API to retry a failed stage: [retryPipeline](htt
...
+
+#### Would like to know what are the account using older delegate version?
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6914]:
```suggestion
#### What accounts are using the older delegate version?
```

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 6]
```diff
@@ -6872,3 +6872,115 @@ Users can utilize the following API to retry a failed stage: [retryPipeline](htt
...
+You can determine if an account is using an older delegate version by checking the Delegate release notes. The release notes provide information about features that require a specific delegate version. If the account is using a version older than the required version for a specific feature, then they are using an older delegate version.
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6915]:
```suggestion
You can determine if an account is using an older delegate version by checking the delegate release notes. The release notes provide information about features that require a specific delegate version. If the account is using a version older than the required version for a specific feature, then they are using an older delegate version.
```

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 7]
```diff
@@ -6872,3 +6872,115 @@ Users can utilize the following API to retry a failed stage: [retryPipeline](htt
...
+
+You can also use check article here to verify which is the supported version for your account and check delegate version which you are using. https://developer.harness.io/docs/platform/delegates/install-delegates/delegate-upgrades-and-expiration/#use-automatic-upgrade-with-custom-delegate-images
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6917]:
```suggestion
To verify what version is supported for your account, and to check which delegate version you are using, go to [Delegate Upgrades and Expiration]9https://developer.harness.io/docs/platform/delegates/install-delegates/delegate-upgrades-and-expiration/#use-automatic-upgrade-with-custom-delegate-images)
```

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 8]
```diff
@@ -6872,3 +6872,115 @@ Users can utilize the following API to retry a failed stage: [retryPipeline](htt
...
+
+Ensure you have a role with permissions to view, create/edit, and delete roles, such as Account Admin.
+Navigate to the scope where you want to create the role:
+For account scope: Go to Account Settings > Access Control.
+For organization scope: Go to Account Settings > Organizations > Select the relevant organization > Access Control.
+For project scope: Go to Projects > Select the relevant project > Access Control.
+Select "Roles" in the header, then choose "New Role".
+Enter a name for the role, and optionally provide a description and tags.
+Click "Save" and select permissions for the role.
+Apply the changes.
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6940]: Can you make these steps a numbered list.

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 9]
```diff
@@ -6872,3 +6872,115 @@ Users can utilize the following API to retry a failed stage: [retryPipeline](htt
...
+To assign the "create projects" role in Harness, follow these steps:
+
+Ensure you have a role with permissions to view, create/edit, and delete roles, such as Account Admin.
+Navigate to the scope where you want to create the role:
+For account scope: Go to Account Settings > Access Control.
+For organization scope: Go to Account Settings > Organizations > Select the relevant organization > Access Control.
+For project scope: Go to Projects > Select the relevant project > Access Control.
+Select "Roles" in the header, then choose "New Role".
+Enter a name for the role, and optionally provide a description and tags.
+Click "Save" and select permissions for the role.
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6939]: Use **<example text>** to make a word bold to indicate an in interface option to select.

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 10]
```diff
@@ -6872,3 +6872,115 @@ Users can utilize the following API to retry a failed stage: [retryPipeline](htt
...
+Apply the changes.
+
+#### How can I generate a policy to block deployments when all tests are not successful?
+
+Yes, you can create a policy to block deployments when all tests are not successful. Here's an example policy that can be applied using the On Run event for a pipeline:
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6944]:
```suggestion
Yes, you can create a policy to block deployments when all tests are not successful. Here's an example policy that can be applied using the **On Run** event for a pipeline:
```

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 11]
```diff
@@ -6872,3 +6872,115 @@ Users can utilize the following API to retry a failed stage: [retryPipeline](htt
...
+wget https://releases.hashicorp.com/terraform/0.13.3/terraform_0.13.3_linux_amd64.zip
+apt-get install unzip
+unzip terraform_0.13.3_linux_amd64.zip
+cp terraform /usr/bin/
+terraform --version
+```
+You can modify this script to install a different version of Terraform if needed. Once you have modified the script, add it to the INIT_SCRIPT environment variable in the delegate YAML file. When you deploy the custom image delegate, the script will be executed during initialization and Terraform will be installed on the delegate.
+ 
+I recommend reviewing the document available at:
+https://developer.harness.io/docs/platform/delegates/manage-delegates/build-custom-images-delegate-dockerfile/
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6986]:
```suggestion
To learn more, go to [Build Custom Image Delegates](https://developer.harness.io/docs/platform/delegates/manage-delegates/build-custom-images-delegate-dockerfile/)
```

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 12]
```diff
@@ -6872,3 +6872,115 @@ Users can utilize the following API to retry a failed stage: [retryPipeline](htt
...
+apt-get -y install git
+wget https://releases.hashicorp.com/terraform/0.13.3/terraform_0.13.3_linux_amd64.zip
+apt-get install unzip
+unzip terraform_0.13.3_linux_amd64.zip
+cp terraform /usr/bin/
+terraform --version
+```
+You can modify this script to install a different version of Terraform if needed. Once you have modified the script, add it to the INIT_SCRIPT environment variable in the delegate YAML file. When you deploy the custom image delegate, the script will be executed during initialization and Terraform will be installed on the delegate.
+ 
+I recommend reviewing the document available at:
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6985]:
```suggestion
```

**Review by SushrutHarness — CHANGES_REQUESTED** [review]
Some typos, and formatting changes!

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 13]
```diff
@@ -6872,3 +6872,115 @@ Users can utilize the following API to retry a failed stage: [retryPipeline](htt
...
+    # Show a human-friendly error message
+    msg := sprintf("test step '%s' failed", [step.name])
+}
+```
+
+This policy will deny pipeline execution if any test step fails. You can modify this policy to suit your specific requirements.
+
+####  What is <+infra.tags.endpoint>?
+
+<+infra.tags.endpoint> is not available as a Harness Variable. Refer to the Supported Infra Harness Variable documentation for more details.
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6965]:
```suggestion
`<+infra.tags.endpoint>` is not available as a Harness Variable. Refer to the Supported Infra Harness Variable documentation for more details.
```

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 14]
```diff
@@ -6872,3 +6872,115 @@ Users can utilize the following API to retry a failed stage: [retryPipeline](htt
...
+    # ... where the step failed
+    step.status == "FAILED"
+    # Show a human-friendly error message
+    msg := sprintf("test step '%s' failed", [step.name])
+}
+```
+
+This policy will deny pipeline execution if any test step fails. You can modify this policy to suit your specific requirements.
+
+####  What is <+infra.tags.endpoint>?
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6963]:
```suggestion
####  What is `<+infra.tags.endpoint>`?
```

**Comment by aimurphy** [comment]
These questions are duplicated/moved to PR 6531. Closing here to avoid MC/unnecessary duplication.

**Pull request closed without merging**